File: pr1-java/networking/src/main/java/pr1Java/networking/reflection/datatransfer/UsernameDto.java
package pr1Java.networking.reflection.datatransfer;

import java.io.Serializable;

public class UsernameDto implements Serializable {
    private final String value;

    public UsernameDto(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }

    @Override
    public String toString() {
        return "UsernameDto[" + getValue() + "]";
    }
}

File: pr1-cs/Networking/DataTransfer/UsernameDto.cs
using System;

namespace Networking.DataTransfer
{
    [Serializable]
    public class UsernameDto
    {
        public UsernameDto(string value)
        {
            Value = value;
        }

        public string Value { get; }

        public override string ToString()
        {
            return "UsernameDto[" + Value + "]";
        }
    }
}